src/stories/Introduction.stories.mdx

```diff
@@ -86,126 +89,43 @@ import StackAlt from './assets/stackalt.svg';
       font-size: 14px;
       line-height: 20px;
     }
-
-    .tip {
-      display: inline-block;
-      border-radius: 1em;
-      font-size: 11px;
-      line-height: 12px;
-      font-weight: 700;
-      background: #E7FDD8;
-      color: #66BF3C;
-      padding: 4px 12px;
-      margin-right: 10px;
-      vertical-align: top;
-    }
-
-    .tip-wrapper {
-      font-size: 13px;
-      line-height: 20px;
-      margin-top: 40px;
-      margin-bottom: 40px;
-    }
-
-    .tip-wrapper code {
-      font-size: 12px;
-      display: inline-block;
-    }
   `}
 </style>
 
-# Welcome to Storybook
+<img src={POPOYOKOcover} alt="popoyoko cover" />
 
-Storybook helps you build UI components in isolation from your app's business logic, data, and context.
-That makes it easy to develop hard-to-reach states. Save these UI states as **stories** to revisit during development, testing, or QA.
+### Welcome to the Popoyoko Design System.
 
-Browse example stories now by navigating to them in the sidebar.
-View their code in the `stories` directory to learn how they work.
-We recommend building UIs with a [**component-driven**](https://componentdriven.org) process starting with atomic components and ending with pages.
+What is our Design System ? 
 
-<div className="subheading">Configure</div>
+This is a **modular** component design system in React and Typescript, usable with **tokens** from the figma design system.
```

Comment:
modular => headless
usable => think for **design tokens**

Reply:
Resolved by this [commit](https://github.com/Popoyoko/popoyoko-ui/pull/201/commits/e0da8ff343f276f1ba43af0343e9ce441368d79d).